src/twist_mux.cpp

```diff
@@ -80,9 +83,15 @@ void TwistMux::updateDiagnostics(const ros::TimerEvent& event)
   diagnostics_->updateStatus(status_);
 }
 
-void TwistMux::publishTwist(const geometry_msgs::TwistConstPtr& msg)
+void TwistMux::publishTwist(const geometry_msgs::TwistConstPtr& msg, const std::string& name)
 {
   cmd_pub_.publish(*msg);
+  if(pub_status_continuously_ || name != last_pub_name_){
```

Comment:
That mostly had to do with my application at the moment of writing this. I had it going through a ROS1 bridge to ROS2 which meant latching did not work for me so I had to get around it with that solution. I can probably remove it now since that specific application is no longer something I need.

Reply:
Thanks, it's probably nicer not to modify all nodes for a shortcoming in
the bridge :)